staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -217,9 +233,12 @@ func NewObjectTracker(scheme ObjectScheme, decoder runtime.Decoder) ObjectTracke
 		scheme:   scheme,
 		decoder:  decoder,
 		objects:  make(map[schema.GroupVersionResource]map[types.NamespacedName]runtime.Object),
-		watchers: make(map[schema.GroupVersionResource]map[string][]*watch.RaceFreeFakeWatcher),
+		watchers: make(map[schema.GroupVersionResource]map[string][]Watcher),
 	}
 }
+func (t *tracker) ExchangeWatcher(watcher ExchangeWatcher) {
```

Comment:
re: other comment about ExchangeWatcher

This would go away if using the `ObjectTrackerOptions` method.